ghost/audience-feedback/lib/AudienceFeedbackService.js

```diff
@@ -1,7 +1,21 @@
+const urlUtils = require('../../core/core/shared/url-utils');
```

Comment:
Would be cool to pass in the necessary data as dependencies, similar to how we do here with `baseUrl` https://github.com/TryGhost/Ghost/blob/main/ghost/link-redirects/lib/LinkRedirectsService.js

Reply:
Fixed :)